.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -101,4 +101,4 @@
       <option name="ignoreNonEmtpyLoops" value="false" />
     </inspection_tool>
   </profile>
-</component>
\ No newline at end of file
+</component>
```

Comment:
I think intellij always saves its xml files without trailing newline, so this would be changed back whenever the inspection profile is edited again.